app/views/shared/_modal_layout.html.erb

```diff
@@ -1,6 +1,11 @@
 <div class="modal-backdrop">
 </div>
-<div class="padding-x-2 padding-y-6 modal" role="dialog">
+<div   
+  aria-describedby="dialog_description"
+  aria-labelledby="dialog_label"
```

Comment:
There appear to be a few other partials which use this layout, but which don't also have label and description elements with these IDs. Do we need to add those as well?

https://github.com/18F/identity-idp/blob/c747e4924172d46d4e41160f7fcf5e5404598726/app/views/reactivate_account/_modal.html.erb#L2

https://github.com/18F/identity-idp/blob/c747e4924172d46d4e41160f7fcf5e5404598726/app/views/session_timeout/_warning.html.erb#L2

Reply:
Based on my understanding so far, we will have to add a label and description to each modal. Since those pieces are dynamic and will not conflict with each other, I _think_ we can use the same description and label for the other modals. I am going to make sure my understanding is correct on this.